### ISSUE TITLE
Make the "depot" default for transit models

### ISSUE THREAD

**Issue by rikardn** [pharmpy/modelbuilder#129]
I think this is the most natural first step if going to transits.

**Comment by rikardn** [comment]
At least when you have FO absorption.

**Comment by antoniaih** [comment]
Depot is set as default now for all types of absorption. Or should it only be default for FO absorption?

**Comment by rikardn** [comment]
I think we have another issue mentioning some combinations that we don't allow. For example ZO+transits and SEQ+transits.

**Comment by antoniaih** [comment]
Yes, SEQ+transits without depot gives an error. ZO+transits works.

**Comment by antoniaih** [comment]
I blocked ZO with 1 transit and SEQ-ZO-FO with transit without depot. But maybe this issue needs to be dicussed later.

**Comment by rikardn** [comment]
Ok. Yes I think we'll need to go through all cases for absorption rate and delay and what makes sense to not allow.
